tabs/tab_logit.py

```
@@ -311,25 +319,82 @@ def _update_inputs():
         # Update Tab 2 Inputs
+        # (Keep binary for subgroup outcomes usually, unless we expand subgroup module too)
+        binary_cols = [c for c in cols if d[c].nunique() == 2]
         ui.update_select("sg_outcome", choices=binary_cols)
         ui.update_select("sg_treatment", choices=cols)
         ui.update_select("sg_subgroup", choices=sg_cols)
         ui.update_selectize("sg_adjust", choices=cols)
 
+    # --- Interaction Logic ---
+    @reactive.Effect
+    @reactive.event(input.btn_add_interaction)
+    def _add_interaction():
```

Ruff (0.14.10)

Missing return type annotation for private function `_add_interaction`

Add return type annotation: `None`

(ANN202)